src/goto-programs/remove_virtual_functions.cpp

```diff
@@ -280,12 +337,15 @@ void remove_virtual_functionst::get_child_functions_rec(
     }
     functions.push_back(function);
 
-    get_child_functions_rec(
-      child,
-      function.symbol_expr,
-      component_name,
-      functions,
-      visited);
+    if(recurse)
+    {
+      get_parent_functions(
```

Comment:
As above, get_parent_functions is probably a duplicate of the existing resolve-concrete method.